src/app/projects/components/project-metadata-form/project-metadata-form.component.ts

```diff
@@ -124,7 +124,13 @@ export class ProjectMetadataFormComponent implements OnInit, OnDestroy {
 
   onSave({ valid, validationSkipped, value }) {
     if (!this.incrementProjectTab()) {
-      if (valid || validationSkipped) {
+      if (valid) {
+        this.saveProject(value);
+      } else if (validationSkipped) {
+        if (this.create) {
+          value.isInCatalogue = false;
+          alert('This invalid project will be saved, but has been removed from the project catalogue.');
```

Comment:
When do we use `alert()` and when do we use `this.alertService`?

Reply:
I'm using `alert` in this case because I can't use `this.alertService`, the save clears the alert service and triggers a page navigation, which resets the alert service.